src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/AspNet/ApiController_ExecuteAsync_Integration.cs

```diff
@@ -120,7 +120,7 @@ public class ApiController_ExecuteAsync_Integration
                 {
                     scope.Span.SetHeaderTags<IHeadersCollection>(httpContext.Response.Headers.Wrap(), Tracer.Instance.Settings.HeaderTags, defaultTagPrefix: SpanContextPropagator.HttpResponseHeadersTagPrefix);
                 }
-
+                HttpContextHelpers.AddHeaderTagsFromHttpResponse(httpContext, scope);
```

Comment:
Shouldn't this new line replace the few lines above?

Reply:
Yep you're right, fixed in f9debd5b8032b91f44e8d852ac9f8f7c475cec3a